README.md

```diff
@@ -2,208 +2,394 @@
 
 ## Learning Goals
 
-- Explain in general terms what the execution context is.
-- Describe the difference between global- and function-scoped code.
-- Understand how block scoping affects variables declared with `let` and `const`.
+- Define scope as a programming concept.
+- Describe the three levels of scope in JavaScript.
+- Examine the exceptions to the scope rules.
 
 ## Introduction
 
-_Scope_ is, in short, the concept of **where something is available**. In the
-case of JavaScript, it has to do with where declared variables and methods are
-available within our code.
+The more complex your code becomes, the more important it is to pay attention to
+scope. So, what exactly is it? _Scope_ is the concept of **where something is
+available**. In the case of JavaScript, it has to do with where variables and
+functions are accessible within our code.
 
-Scope is a ubiquitous concept in programming and one of the most misunderstood
-principles in JavaScript, frustrating even seasoned engineers. Not understanding
-how scope works will lead to pain.
+Scope is not something fully tangible, there is no specific syntax to it. As a
```

Comment:
```suggestion
Scope is not a tangible "thing" - it doesn't have a specific syntax, or even a specific value. It's more of a rule that _applies_ to our code, rather than a specific _piece_ of our code. As a
```